src/kvstore/DiskManager.h

```diff
@@ -15,11 +15,14 @@
 #include "common/base/StatusOr.h"
 #include "common/thread/GenericWorker.h"
 #include "common/thrift/ThriftTypes.h"
+#include "interface/gen-cpp2/meta_types.h"
 
 namespace nebula {
 namespace kvstore {
 
 using PartDiskMap = std::unordered_map<std::string, std::set<PartitionID>>;
+using DiskPaths =
```

Comment:
why use thrift structure？

Reply:
to make serialize easier, use apache::thrift::CompactSerializer::serializer to generate the value.

Reply:
actually it's same with `PartDiskMap`